.ci/scripts/lib/flaky-tests.sh

```diff
@@ -1,17 +1,18 @@
 #!/bin/bash
 
 #######################################
-# Analyses flaky tests produced by flaky-test-extractor-maven-plugin
+# Parses Maven output to find if any tests was run more than once.
```

Comment:
I'm guessing this was unintentional? :smile:

Reply:
:facepalm: yup, thanks! I still need to test the artifact archival anyway though.